.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: build
+on:
+  push:
+    branches:
+      - master
+      - /\d\.0\.0-RC/
+  pull_request:
+    branches:
+      - master
+      - /\d\.0\.0-RC/
+jobs:
+  check-coverage:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
I miss dependencies cache - see https://docs.github.com/en/actions/guides/building-and-testing-java-with-maven#caching-dependencies